systemtest/build.xml

```diff
@@ -90,22 +88,15 @@
 				</exec>
 			</then>
 		</if>
-		<if>
-			<contains string="${JVM_VERSION}" substring="openj9"/>
```

Comment:
Understand that this is redundant as that target is already called on line 158.

Reply:
Hi @ShelleyLambert , yes. The redundant call was removed.